src/main/resources/language/German.properties

```diff
@@ -2259,4 +2261,21 @@ ls.player.lastlineup = Position letzter Einsatz
 ls.player.motherclub.name = Heimatverein
 ls.player.matchescurrentteam = Tore für aktuellen Verein
 ls.about.database.folder = Datenbank-Verzeichnis
-ls.about.logs.folder = Logging-Verzeichnis
\ No newline at end of file
+ls.about.logs.folder = Logging-Verzeichnis
```

Comment:
@wsbrenk Have we moved away from poeditor?

Reply:
I am not @wsbrenk but I will answer:

No, I think not.

I wanted to persist the change and made the change already additionally in POEditor for Spanish.

This PR has to be deleted if it works.

Reply:
If I remember correctly (and @wsbrenk, keep me honest here!), new strings get added to `English.properties`, which then gets published to the poeditor website.

Translators go into the poeditor website, and provide the translations for their respective language there.  At build time, HO then pulls the translation files along with the new translations, and package them along with the rest of the code.

So if you need to add/change a German translation, this needs to be done on poeditor website.

There were however recently issues with that project, and @wsbrenk  floated the idea of changing the translation files directly in Github – which is not great for translators, who may not be tech savvy.

Hope this makes sense!

Reply:
@tychobrailleur decribes the translation workflow correctly. Again i tried to contact akasolace asking him to move the poeditor project. otherwise it will be archived again very soon. In the meantime, it occurred to me that we could create a new project at poeditor and import the property files of the current project there. The translators would then all have to register again.